kaggler/preprocessing/categorical.py

```diff
@@ -547,3 +547,110 @@ def transform(self, X):
     def fit_transform(self, X, y):
         self.fit(X, y)
         return self.transform(X)
+
+
+class FrequencyEncoder(base.BaseEstimator):
+    """Frequency Encoder that encode categorical values by counting frequencies.
+
+    Attributes:
+        frequency_encoders (list of dict): frequency encoders for columns
+    """
+
+    def __init__(self, cv=None):
+        """Initialize the FrequencyEncoder class object.
+        Args:
+            cv (sklearn.model_selection._BaseKFold, optional): sklearn CV object
+        """
+        self.cv = cv
+
+    def __repr__(self):
+        return('FrequencyEncoder(cv={})'.format(self.cv))
+
+    def _get_frequency_encoder(self, x, y):
```

Comment:
We don't need/use `y` here. Let's remove it.

Reply:
yeah, thanks for pointing it out!